build/webpack.client.dev.js

```diff
@@ -63,14 +54,7 @@ const devSpecificConfig = {
     }),
   ],
   resolve: {
-    alias: {
-      ...relativeClientAliases,
-    },
-    plugins: [
-      withTsconfigPathsPlugin({
-        configFile: './client/tsconfig.json',
-      }),
-    ],
+    plugins: [withTsconfigPathsPlugin({ configFile: 'client/tsconfig.json' })],
```

Comment:
Nothing using the relative aliases yet. And if they later do, they'll go into tsconfig now.